### PR TITLE
ButtonGroup Refactor

### PR DESCRIPTION
**Pull request by Gabojcb**
Hi development team!

I have created this pull request to share with you a refactoring and improvement of the Button Group component. I have added the orientation and size attributes to provide more flexibility in its use and made improvements to the styles for a more appealing appearance.

I hope you find these changes useful and I am eager to receive your feedback.

Thank you for your time and consideration!

Best regards,
Gabojcb

### CODE REVIEW

File: docs/modules/components/tooltips/module.json
Comment (jircdev) [inline docs/modules/components/tooltips/module.json:1]: I think the paths can have errors.
The path of `ts` processor is not the same that `mdx` path @Gabojcb

File: docs/modules/layouts/main/ts/menu/index.ts
Comment (jircdev) [inline docs/modules/layouts/main/ts/menu/index.ts:1]: Please validate your Prettier configuration. Our code standard requires the use of single quotes, which isn't followed in this code.

File: project/modules/components/scss/button-group/size.scss
Comment (jircdev) [inline project/modules/components/scss/button-group/size.scss:1]: Buttons should not have a directly set font-size; this value should be defined in the typography section. Please remove it, or if needed, separate this change for discussion.

**Review by jircdev — COMMENTED** [review]
Kindly review the observations and inform me for further validation. Your input is greatly appreciated.

File: docs/modules/components/tooltips/module.json
Comment (Gabojcb) [reply docs/modules/components/tooltips/module.json:1]: Thansk for the observations, i now ajustment changes

File: project/modules/components/ts/button-group/button-group.tsx
Comment (jircdev) [inline project/modules/components/ts/button-group/button-group.tsx:1]: Sure, here's a review comment you can use to explain the bracket spacing rule:

"Hi @Gabojcb 

Thank you for your pull request! I noticed a small inconsistency regarding the spacing around brackets. According to our coding standard and Prettier configuration, we aim for consistent code formatting.

To ensure adherence, could you please adjust the spacing within brackets in lines like:

Before:
```javascript
import type {IProps} from './type';
```

After:
```javascript
import * as React from "react";
import type { IProps } from "./type";
```

You can update our `.prettierrc` configuration file to handle this automatically by setting `"bracketSpacing": true`.

Let me know if you have any questions or if there's anything I can assist you with.

Best regards,

Comment (Gabojcb) [reply project/modules/components/ts/button-group/button-group.tsx:1]: It's perfect, I didn't take it into account, I'll make the changes.

**Review by jircdev — APPROVED** [review]

**Pull request merged**